Blockzilla/URLBar.swift

```diff
@@ -662,17 +664,21 @@ class URLBar: UIView {
     }
 
     fileprivate func setTextToURL() {
-        var fullUrl: String? = nil
+        var displayText: String? = nil
         var truncatedURL: String? = nil
 
         if let url = url {
             // Strip the username/password to prevent domain spoofing.
             var components = URLComponents(url: url, resolvingAgainstBaseURL: false)
             components?.user = nil
             components?.password = nil
-            fullUrl = components?.url?.absoluteString
+            
+            if let searchedText = SearchHistoryUtils.pullSearchFromStack(), searchedText != "" {
```

Comment:
After you change the function to return nil, you should be able to remove the check for != ""